mise/tasks/test/daemon.sh

```
@@ -0,0 +1,5 @@
+#\!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)